### DISCUSSION TITLE
Purchase Price for Income Losses is nill

### DISCUSSION THREAD

**Discussion by BienBrian** [Blair2004/NexoPOS#982] [Ideas]
The purchase price in the Incomes & Losses table is zero for all products despite the products having all their prices. 

Kindly assist

**Comment by Blair2004** [comment]
Hi, 
If you're using NexoPOS 4.8.1 you can restore purchase price like this (with a CLI command):

```
php artisan ns:doctor --fix-orders-products
```
This command will pull purchase price for each product and add it to each sold products for the report.

**Comment by Blair2004** [comment]
Any update ?

**Comment by BienBrian** [comment]
I was not aware that 4.8.1 was out already.

Let me first update from 4.8.0 and check if my problem is resolved.

Thanks

**Comment by BienBrian** [comment]
Hello, the CLI command added the purchase price to the table. 

However, there is something strange I saw plus a few more bugs. I'm dropping the screenshots in a minute

**Comment by BienBrian** [comment]
![image](https://user-images.githubusercontent.com/33430455/182899180-b6d27720-55b7-462f-88a4-ebe58c19d6ba.png)
I noticed that some products were selling for a loss. 
Please add a setting to ensure that sale price is never below purchase price, if it is possible

**Comment by BienBrian** [comment]
Also, could you please make the header and footer for this table Sticky? When one has many rows, the user experience is affected.

**Comment by Blair2004** [comment]
Hi, we'll consider those as possible improvement for those reports.

**Comment by Blair2004** [comment]
Hi, quick update on this. The issue has been fixed now.
